### PR TITLE
Blog Update: new languages, fixes

### PR DESCRIPTION
**Pull request by STCB**
Translated the Peach Blog in all available languages, blog display fixes

### CODE REVIEW

**Comment by learntheropes** [comment]
This pull request introduced a bug that needs to be addressed: the English blog page returns a 404. 
We are investigating the changes.

File: content/de/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
```
Comment (MManke188) [inline content/de/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md:1]: The name of this file is french

File: content/el/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (MManke188) [inline content/el/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md:1]: Again a french file name

File: content/es/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
```
Comment (MManke188) [inline content/es/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md:1]: Will stop repeating myself - this blog seems french everywhere

File: content/pl/blog/2023-11-23-why-bitcoin.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (MManke188) [inline content/pl/blog/2023-11-23-why-bitcoin.md:1]: Still english

**Review by MManke188 — COMMENTED** [review]
TBD if we should fix these

File: content/pl/blog/2023-11-23-why-bitcoin.md (hunk 4 above)
Comment (learntheropes) [reply content/pl/blog/2023-11-23-why-bitcoin.md:1]: fixed

File: content/el/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md (hunk 2 above)
Comment (learntheropes) [reply content/el/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md]: fixed

File: content/de/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md (hunk 1 above)
Comment (learntheropes) [reply content/de/blog/2023-09-02-Peach-Bitcoin-fête-ses-un-an.md]: fixed

File: content/de/blog/2023-09-18-1y-anniversary.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
+---
+keywords:
```
[the remark this answers is not in the document]
Comment (learntheropes) [reply content/de/blog/2023-09-18-1y-anniversary.md:2]: fixed

File: content/de/blog/2023-03-06-peach-for-meetups.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
+---
```
[the remark this answers is not in the document]
Comment (learntheropes) [reply content/de/blog/2023-03-06-peach-for-meetups.md:1]: fixed

**Review by MManke188 — APPROVED** [review]

**Pull request merged**